Decisions.Equifax/ConsumerCreditReport/ConsumerCreditReportSteps.cs

```diff
@@ -1,162 +1,27 @@
-﻿using System;
-using System.IO;
-using System.Net;
-using System.Text.Encodings.Web;
-using Decisions.Equifax.ConsumerCreditReport.Request;
+﻿using Decisions.Equifax.ConsumerCreditReport.Request;
 using Decisions.Equifax.ConsumerCreditReport.Response;
 using DecisionsFramework;
-using DecisionsFramework.Design.ConfigurationStorage.Attributes;
 using DecisionsFramework.Design.Flow;
-using DecisionsFramework.Design.Flow.CoreSteps;
-using DecisionsFramework.Design.Flow.Mapping;
-using DecisionsFramework.Design.Properties.Attributes;
 using DecisionsFramework.ServiceLayer;
-using DecisionsFramework.ServiceLayer.Services.ContextData;
-using Newtonsoft.Json;
 
 namespace Decisions.Equifax.ConsumerCreditReport
 {
     [AutoRegisterMethodsOnClass(true, "Integration/Equifax/Consumer Credit Report")]
     public class ConsumerCreditReportSteps
     {
-        private static readonly Log log = new Log(EquifaxConstants.LogCat);
-        
         /// <summary>
         /// Step: Get Consumer Credit Report (Integration/Equifax/Consumer Credit Report/Get Consumer Credit Report)
         /// </summary>
         public static ConsumerCreditReportResponse GetConsumerCreditReport(ConsumerCreditReportRequest request)
         {
-            return ExecuteCreditReportRequest(request);
-        }
-
-
-
-        /// <summary>
-        /// Helper: Execute API Endpoint
-        /// </summary>
-        private static ConsumerCreditReportResponse ExecuteCreditReportRequest(ConsumerCreditReportRequest request)
-        {
-            JsonSerializerSettings jsonSettings = new JsonSerializerSettings
+            string scope = ModuleSettingsAccessor<EquifaxSettings>.GetSettings().EquifaxConsumerCreditReportScope;
+            string requestUrl = ModuleSettingsAccessor<EquifaxSettings>.GetSettings().EquifaxConsumerCreditReportEndpoint;
```

Comment:
You can get the settings once and store it in a variable then use that to get the scope and endpoint.